filebeat/autodiscover/builder/hints/config.go

```diff
@@ -28,8 +28,10 @@ func defaultConfig() config {
 	rawCfg := map[string]interface{}{
 		"type": "docker",
 		"containers": map[string]interface{}{
-			"ids": []string{
-				"${data.container.id}",
+			"paths": []string{
+				// To be able to use this builder with CRI-O replace paths with:
+				// /var/log/pods/${data.kubernetes.pod.uid}/${data.kubernetes.container.name}/*.log
+				"/var/lib/docker/containers/${data.kubernetes.container.id}/*-json.log",
```

Comment:
Would this path need to be added to the annotations of any container using CRI-O runtime?

Maybe this is not practical for clusters using this runtime. We could think on providing different defaults depending on the runtime, but this can be done as a follow up.

Let me know what you think about this.

Reply:
container IDs are already set in the metadata along with the runtime information. All the metadata required to enable CRI-O are already present in the current state. The ID does not vary based on runtime as the ID used by all of them is the cgroup id.

Reply:
I was thinking on how to use this with CRI-O and hints-based autodiscover, I see the information is there, but as defaults are for docker, I wonder if when using hints-based autodiscover we'll have to add this config to all pods.
In any case if it works as before I am ok with continuing with this and thinking on hints-based autodiscover in follow ups.

Regarding working as before, I have just noticed that we are using `data.kubernetes.container.id` now, but I think this builder is also used by docker provider, we should keep using the generic variables (`data.container.id`), any reason to change it to the kubernetes ones?